ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -70,13 +71,17 @@ for i in "${!DATACENTERS[@]}"; do
         wait "$!"
     fi
 
-    hw_versions=(15 17 18 19)
+    hw_versions=(15 17 18 19 20)
     for hw_version in "${hw_versions[@]}"; do
```

Comment:
maybe I wasn't paying attention to a previous pr
but does this import an ova per datacenter per version? just make sure I am parsing that right

Reply:
yeah, to test vsphere UPI zonal deployment, see https://github.com/openshift/release/pull/35175/